eng/Packages.Data.props

```diff
@@ -122,7 +122,6 @@
     <!-- OpenTelemetry dependency approved for Azure.Monitor.OpenTelemetry.Exporter package only -->
     <PackageReference Update="OpenTelemetry" Version="1.4.0" />
     <PackageReference Update="OpenTelemetry.Exporter.InMemory" Version="1.4.0" />
-    <PackageReference Update="OpenTelemetry.Extensions.AzureMonitor" Version="1.0.0-beta.4" />
```

Comment:
what is the reasoning behind this change?

Reply:
Plan is to deprecate the package and move those changes here.
I'll ping @rajkumar-rangaraj to discuss with you.

Reply:
The `OpenTelemetry.Extensions.AzureMonitor` package has been customized specifically for the Azure Monitor Exporter and is exclusively used by this project. In order to reduce maintenance overhead and simplify configuration for customers, we have moved the implementation to this repository. As part of this change, we plan to remove the code from the contrib repository and deprecate the `OpenTelemetry.Extensions.AzureMonitor` NuGet package.